lib/model/maalcie/MaaltijdAbonnementenModel.class.php

```diff
@@ -154,39 +149,14 @@ private static function loadLedenAbonnementen($alleenNovieten = false, $alleenWa
 	}
 
 	public static function getAbonnementenVoorRepetitie($mrid) {
-		return self::loadAbonnementen($mrid);
+        return static::instance()->find('mlt_repetitie_id = ?', array($mrid));
 	}
 
 	public static function getAbonnementenVanNovieten() {
 		$matrix_repetities = self::getAbonnementenMatrix(true);
 		return $matrix_repetities[0];
 	}
 
-	/**
-	 * Laad abonnementen van een bepaalde repetitie OF voor een bepaald lid.
-	 * 
-	 * @param int $mrid
-	 * @param String $uid
-	 * @return MaaltijdAbonnement[]
-	 */
-	private static function loadAbonnementen($mrid = null, $uid = null) {
```

Comment:
Dit is ook zo'n functie die op tienduizend verschillende manieren kan worden gebruikt. Heb je alle nodige use-cases afgedekt?